src/components/SelectComponents/Autosuggest/SuggestionTag/__tests__/SuggestionTag.spec.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
+import { SuggestionTag } from '../SuggestionTag';
+
+describe('SuggestionTag', () => {
+    it('should render nothing if no children are provided', () => {
+        const view = render(<SuggestionTag />);
+
+        expect(view.baseElement).toMatchSnapshot();
+        expect(view.container.children).toHaveLength(0);
```

Comment:
Here I think looks more elegant to use `.toBeEmptyDOMElement()`

Reply:
Children return array when i use this function it shows
![image](https://github.com/textkernel/oneui/assets/104063210/6306cf5f-ef98-4b2d-b135-32ec3c2853f7)

Reply:
Okay i get it